test/functional/lib/ipfs-request-gateway-redirect.test.js

```diff
@@ -275,43 +386,63 @@ describe('modifyRequest.onBeforeRequest:', function () {
         // or could produce false-positives such as redirection from localhost:5001/ipfs/path to localhost:8080/ipfs/path
         it('should fix localhost Kubo RPC hostname to IP', async function () {
           const request = url2request('http://localhost:5001/ipfs/QmPhnvn747LqwPYMJmQVorMaGbMSgA7mRRoyyZYz3DoZRQ/')
-          // expectNoRedirect(modifyRequest, request)
-          expect((await modifyRequest.onBeforeRequest(request)).redirectUrl)
-            .to.equal('http://127.0.0.1:5001/ipfs/QmPhnvn747LqwPYMJmQVorMaGbMSgA7mRRoyyZYz3DoZRQ/')
+
+          if (!isManifestV3) {
+            // this is set as a default rule in MV3
```

Comment:
Can we add a test for that? `DEFAULT_LOCAL_RULES` seems to be only used in `reconcileRulesAndRemoveOld` so maybe add unit test for that?

(Not blocker, ok to fill issue and do it as followup)